tree.go

```diff
@@ -710,7 +715,11 @@ func (n *InternalNode) Commit() *Point {
 				wg.Add(1)
 				go func() {
 					defer wg.Done()
-					commitNodesAtLevel(nodes[start:end])
+					if err := commitNodesAtLevel(nodes[start:end]); err != nil {
+						// TODO: make Commit() return an error
+						panic(err)
+					}
```

Comment:
@gballet, we have some tension here. `commitNodesAtLevel` error was ignored, but actually we should check it. But that would mean bubbling the error to `Commit()`.

For now, considering the work that we're doing for Kaustinen, we can panic here for a while. But at some point I think there won't be other option that bubbling the error.

There's already an issue to resolve all the panics in the repo, so this will be part of that work.

WDYT?

Reply:
Not a big fan of %w unless we're sure we're OK with exposing the error as part of the API. But maybe that's the case.